lib/cursor.mongodb.js

```diff
@@ -1,12 +1,14 @@
 'use strict';
 
-const util = require('util');
+const api = {};
+api.util = require('util');
+
 const transformations = require('./transformations.js');
```

Comment:
...but in this certain place `transformations` is a part of current module, internal/submodule, no need to put it inside api. I'll put external requires into api.* for consistence with Impress namespaces.